wolfcrypt/src/integer.c

```diff
@@ -4729,12 +4729,9 @@ LBL_U:mp_clear (&v);
 #endif /* WOLFSSL_KEY_GEN */
 
 
-#if defined(HAVE_ECC) || defined(WOLFSSL_KEY_GEN) || defined(HAVE_COMP_KEY)
```

Comment:
mp_s_rmap is only used if HAVE_ECC is defined.
Some compilers complain about unused static const data and others don't.

Reply:
Could you try running:

./configure --disable-ecc --enable-debug
./configure --disable-ecc --disable-fastmath --enable-debug

These are used when HAVE_ECC is not defined and debugging is enabled.